README.rst

```diff
@@ -133,6 +133,37 @@ and append ``YOUR_FAS_USERNAME@fedoraproject.org`` to the ``tahrir.admin`` optio
 When you login, you should now be able to see the admin view of tahrir in your
 local copy at http://localhost:8000/admin.
 
+Hacking
+=======
+
+Hacking with Vagrant
+--------------------
+Quickly start hacking on tahrir using the vagrant setup that is included in the
+tahrir repo is super simple.
+
+First, install Ansible, Vagrant, the vagrant-sshfs plugin, and the vagrant-libvirt
+plugin from the official Fedora repos::
+
+    $ sudo dnf install ansible vagrant vagrant-libvirt vagrant-sshfs
+
+
+Now, from within main directory (the one with the Vagrantfile in it) of your git
+checkout of tahrir, copy the Vagrantfile.example file to Vagrantfile:
+
+   $ cp Vagrantfile.example Vagrantfile
+
+Run the ``vagrant up`` command to provision your dev environment::
+
+    $ vagrant up
+
+When this command is completed (it may take a while) start tahrir with the
+following command:
+
+    $ vagrant ssh -c"cd /vagrant/; pserve --reload development.ini"
```

Comment:
I recently converted the Bodhi Vagrant environment to put a systemd unit file on the box so that Bodhi is automatically run for you. It's pretty sweet, and it means the user doesn't have to do the above step. Here's some links to the bits that do this:

https://github.com/fedora-infra/bodhi/blob/2.4.0/devel/ansible/roles/dev/files/bodhi.service
https://github.com/fedora-infra/bodhi/blob/2.4.0/devel/ansible/roles/dev/tasks/main.yml#L103-L107
https://github.com/fedora-infra/bodhi/blob/2.4.0/devel/ansible/roles/dev/tasks/main.yml#L123-L127

It's just a suggestion, but I think it's a nice finishing touch.